openassessment/assessment/errors/staff.py

```diff
@@ -0,0 +1,33 @@
+"""
+Errors for the staff assessment api.
+"""
+
+
+class StaffAssessmentError(Exception):
+    """Generic Staff Assessment Error
+
+    Raised when an error occurs while processing a request related to
+    staff assessment.
+
+    """
+    pass
+
+
+class StaffAssessmentRequestError(StaffAssessmentError):
+    """Error indicating insufficient or incorrect parameters in the request.
+
+    Raised when the request does not contain enough information, or incorrect
+    information which does not allow the request to be processed.
+
+    """
+    pass
+
+
+class StaffAssessmentInternalError(StaffAssessmentError):
```

Comment:
It seems a little unnecessary to me to have a unique error class for this. I see that the code already does this, but I think having a single class of ``AssessmentInternalError`` would be simpler. Does the code ever need to distinguish between different kinds of internal errors?